.circleci/config.yml

```diff
@@ -161,4 +161,4 @@ workflows:
             - build_for_tests
       - journey_tests_firefox:
           requires:
-            - build_for_tests
+            - journey_tests_chrome
```

Comment:
we are testing chrome engine on firefox? I don't get this.

Reply:
This just defines the work flow. It means that it requires the journey tests for chrome to be completed before moving on to the firefox tests.

Reply:
got it